fluent/fluent_test.go

```diff
@@ -248,6 +268,51 @@ func TestAsyncConnect(t *testing.T) {
 	}
 }
 
+func Test_PostWithTime(t *testing.T) {
```

Comment:
Indeed, `Test_PostWithTimeNotTimeOut` seems to be more appropriate to express what its testing. I renamed it 428fb85.